src/reports.rs

```diff
@@ -176,7 +176,7 @@ impl Reporter {
                 "graph_env": &self.graph_env,
                 "timestamp": timestamp,
                 "api_key": &client_request.api_key,
-                "user_address": &client_request.user_address,
+                "user": &client_request.user,
```

Comment:
The field name here must not change, since this is the kafka message output. Changing it would break consumers of the topic.

Reply:
I reached out on slack regarding renaming, but to follow up here too, it makes more sense for us to make the renaming consistently for all of the components of the gateway, rather than a partial renaming. That would include renaming in api key service and consumers of the kafka messages

Reply:
Agreed, though we cannot merge changes to external-facing names until significant progress is made in #966.